src/screens/novel/components/EpubIconButton.tsx

```diff
@@ -0,0 +1,156 @@
+import * as React from 'react';
+import useBoolean from '@hooks/useBoolean';
+import { IconButton, Portal } from 'react-native-paper';
+import ChooseEpubLocationModal from './Modal/ChooseEpubLocationModal';
+import { Dimensions, StatusBar, StyleProp } from 'react-native';
+import { IconProps } from 'react-native-paper/lib/typescript/src/components/MaterialCommunityIcon';
+import { ThemeColors } from '@theme/types';
+
+import EpubBuilder from '@cd-z/react-native-epub-creator';
+import { ChapterItem, LibraryNovelInfo } from '@database/types';
+import { showToast } from '@hooks/showToast';
+import { getChapterFromDb } from '@database/queries/DownloadQueries';
+import { useReaderSettings } from '@redux/hooks';
+
+interface extendedNovelInfo extends LibraryNovelInfo {
+  chapters: ChapterItem[];
+}
+interface EpubIconButtonProps {
+  theme: ThemeColors;
+  style: StyleProp<IconProps>;
+  novel: extendedNovelInfo;
+}
+
+const EpubIconButton: React.FC<EpubIconButtonProps> = ({
+  theme,
+  style,
+  novel,
+}) => {
+  const chooseEpubLocationModal = useBoolean(false);
+  const readerSettings = useReaderSettings();
+  const layoutHeight = Dimensions.get('window').height;
+  const epubStyle = `html {
+    scroll-behavior: smooth;
+    overflow-x: hidden;
+    padding-top: ${StatusBar.currentHeight};
+    word-wrap: break-word;
+  }
+  body {
+    padding-left: ${readerSettings.padding}%;
+    padding-right: ${readerSettings.padding}%;
+    padding-bottom: 40px;
+    font-size: ${readerSettings.textSize}px;
+    color: ${readerSettings.textColor};
+    text-align: ${readerSettings.textAlign};
+    line-height: ${readerSettings.lineHeight};
+    font-family: "${readerSettings.fontFamily}";
+  }
+  chapter{
+    display: block;
+  }
+  hr {
+    margin-top: 20px;
+    margin-bottom: 20px;
+  }
+  a {
+    color: ${theme.primary};
+  }
+  img {
+    display: block;
+    width: auto;
+    height: auto;
+    max-width: 100%;
+  }
+  
+  .nextButton,
```

Comment:
If we are directly getting the chapter from DB these styles wouldn't be there, right?

Reply:
I don't know what exactly you mean.
- First why shouldn't the styles be there
- and second at the moment this feature only takes the already downloaded chapters from the DB and transforms it into an epub

Reply:
Oh right, I was to lazy to remove the not needed css and later forgot to remove it.